pahfit/model.py

```diff
@@ -308,7 +318,9 @@ def _convert_spec_data(spec, z):
 
         """
         if not spec.flux.unit.is_equivalent(units.intensity):
-            raise PAHFITModelError("For now, PAHFIT only supports intensity units, i.e. convertible to MJy / sr.")
+            raise PAHFITModelError(
+                "For now, PAHFIT only supports intensity units, i.e. convertible to MJy / sr."
+            )
         y = spec.flux.to(units.intensity).value
```

Comment:
Please lets `x` -> `lam` and `y` -> `flux` everywhere.

Reply:
I have done `x` -> `lam_obs`, `y` -> `flux_obs`
To separate it from
`xz` -> `lam`, `yz` -> `flux`

The former is observer frame, the later is rest frame.

Reply:
Thanks.  See also #294; I believe we have the "shift to rest" wrong for the flux.